system/etc/ih8sn_sweet.conf

```diff
@@ -0,0 +1,4 @@
+BUILD_FINGERPRINT=Redmi/sweet_global/sweet:13/TKQ1.221013.002/V14.0.2.0.TKFMIXM:user/release-keys
+BUILD_DESCRIPTION=sweet_global-user 13 TKQ1.221013.002 V14.0.2.0.TKFMIXM release-keys
+BUILD_SECURITY_PATCH_DATE=2023-04-01
+PRODUCT_NAME=sweet
```

Comment:
It should be `sweet_global`

Reply:
> It should be `sweet_global`

I'm using it with just `sweet` and everything seems to be fine here. Should I still change it?

Reply:
Change or remove. You sure it's needed for the device.? Default should be fine.